### ISSUE TITLE
option for max height of parent

### ISSUE THREAD

**Issue by onceatime** [runspired/flexi#44]
if possible can you add a attribute like "fill parent"? this way its possible to create layouts with fixed header and scrollable columns, right now i haven't found a way to realize such a scenario.

for example: 
![screenshot 2016-04-01 19 44 25](https://cloud.githubusercontent.com/assets/7657474/14215781/62ad0d40-f845-11e5-8277-e6944e93b21e.png)

**Comment by runspired** [comment]
What you want is possible, I'll create a demo later.

**Comment by onceatime** [comment]
oh ok many thx, then i have misunderstood something ;-) sorry

**Comment by onceatime** [comment]
any news?

**Comment by mk-conn** [comment]
+1 Would also be interested in a demo :)

**Comment by rwwagner90** [comment]
@onceatime and @mk-conn can you guys elaborate on what you are trying to achieve? I can try to throw something together.

**Comment by xtagon** [comment]
I'd like to figure this out too. My use case is a full-screen app layout, where you scroll individual containers and never the entire page. Here's an example using flexbox directly but not Flexi: http://bl.ocks.org/jfsiii/5380802 (just add `overflow: scroll` to the column containers).

@rwwagner90 or @runspired can either of you suggest how to do this with Flexi? Sounds like it should be simple but I can't seem to get it right.

**Comment by xtagon** [comment]
Solved it! Here's how:

```
html,
body,
.fullscreen,
body.ember-application > div.ember-view
{
  width: 100% !important;
  height: 100% !important;
  margin: 0 !important;
}
```
The `!important` part may not be necessary for you - I had to override something another CSS framework set.

Next wrap your fullscreen stuff with a wrapper div and give it the `.fullscreen` class.

Do not use `<screen>` or `<page>`, only use `<div class="fullscreen">` and then put your boxes in there.

**Comment by rwwagner90** [comment]
Hi @xtagon you have it working with normal flexbox? Should be fairly easy to convert something that works in flexbox to flexi. If it's working as you want in flexbox, let me know.

**Comment by xtagon** [comment]
@rwwagner90 Thanks. I finally got it working by adding `height: 100%` to the box that has `overflow: scroll`. For some reason it doesn't work with just `<box fill>`

**Comment by rwwagner90** [comment]
@xtagon would you be interested in contributing the example you are talking about here to the flexi dummy app, for us to show as an example?

**Comment by xtagon** [comment]
@rwwagner90 Absolutely. My case depends on some quirks with how I'm using Semantic-UI's sidebar pusher in my app, but I should be able to put together a more minimal example.

**Comment by rwwagner90** [comment]
@xtagon great! Please let me know if you need any help figuring out where to put all the stuff for the examples.

**Comment by xtagon** [comment]
@rwwagner90 I can use some help. I looked at recent PRs for the examples and tried to follow that as a guide. I created a route, linked to it in the index, created a template for my layout and a route component with a template. I just copied the SVG and template snipped from another example, but it won't render in my browser so I must have missed something.

You can see what I tried here: https://github.com/xtagon/flexi/commit/b554267d6eb4dadea9ea5dc8398302df20166f95

**Comment by rwwagner90** [comment]
@xtagon I think the issue is your component has no dash in the name. Components must be dasherized.

**Comment by xtagon** [comment]
Opened PR #130
